src/vxworks/mod.rs

```diff
@@ -2094,6 +2138,43 @@ pub fn posix_memalign(
     }
 }
 
+// epoll.h
+// Unfortunately epoll is currently only supported in the VxWorks kernel
```

Comment:
Why are these here then?

Reply:
We are going to support it in user land.

Reply:
How? IIUC you'd need to provide an implementation in user-land for those, and this crate wouldn't be the place to do that (this crate only provides FFI bindings).

Reply:
I will remove this part.